### ISSUE TITLE
App crashes when using a dropdown 

### ISSUE THREAD

**Issue by ramotswa** [nwjs/nw.js#5136]
Hello,
We are currently on version 0.12.3-win32, running on windows 8.1 (32bit) and we want to upgrade to a better nwjs version. 
Any version we have tried above this has the following problem;
In our angluarjs app, running through nwjs, we two drop down HTML components that make the app crash completely. 

I do have a dump file from that crash, but I am hesitant to upload it as I don't know what is in the dump and the application handles confidential data. I haven't managed to be able to use the minidump_stackwalk on that file so I would love some help with that. 

Presumably I can use the stackwalk for osx to read a dump from a windows verison? If so, then my next issue is getting minidump_stackwalk to work. Whenever I try it says; -bash: minidump_stackwalk: command not found. I downloaded the nwjs-sdk-v0.16.0-osx-x64 to get that. 

Any help would be much appreciated.

**Comment by ghostoy** [comment]
@ramotswa minidump_stackwalk can be built from [breakpad](https://chromium.googlesource.com/breakpad/breakpad/). You'll need Cygwin to build it on Windows. Or you can build it on Linux or Mac and decode crash dumps generated on Windows directly.

**Comment by ramotswa** [comment]
So I have been able to replicate my issues with a very simple test case. 
I am only able to replicate this using some SDK versions of nwjs - and specifically the windows 32 bit versions. Also, the issue is with TOUCH, using a mouse to do this is fine, but as soon as you use your fingers something breaks.
For instance; 
On a touch device running 32bit windows, download https://dl.nwjs.io/v0.15.1/nwjs-v0.15.1-win-ia32.zip. 
Once extracted, cmd into that folder and run nw.exe --url=https://jsfiddle.net/vhypmfc8/ 
Touch the dropdown and select an option. On my system the entire nw window crashes. 

I have managed to get it to work using version 15.4 (SDK), but it fails again in v16.0 (SDK).. I haven't tested the latest one yet.
